index.html

```diff
@@ -3,9 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="stylesheets/index.css">
     <link href="https://fonts.googleapis.com/css?family=Fredericka+the+Great&display=swap" rel="stylesheet">
-    <title>New Tab</title>
+    <title>DarkSite</title>
```

Comment:
I don't think it's a good idea to change the title of the page. A new tab should **feel** like a new tab. Seeing the `New Tab` title preserves the familiarity of the page. I suggest you to restore this.

Reply:
Okay will do that!!